warden/src/persistence.rs

```diff
@@ -8,8 +8,13 @@ use uuid::Uuid;
 
 pub struct RangeInfo {
     pub id: Uuid,
+    pub sub_range_id: String,
```

Comment:
what is the sub_range_id?

Reply:
Not needed for this PR or maybe at all. I was thinking that the base ranges would have stable Uuids to identify them and when we split them into subranges, we might need to assign a new id to distinguish them from each other. What did you have mind for the `id` here?

Reply:
Ah. I envision ranges will have between 0-2 parents, so:

0: I'm guessing ranges with 0 parents are what you mean by base ranges?
1: ranges with 1 parents are the result of a range split
2: ranges with 2 parents are the result of a range merge

Reply:
Okay, I am taking it out now. We can discuss more when I get to implementing splits and merges.